src/components/error-message/index.md.njk

```diff
@@ -9,24 +9,43 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-{{ example({group: 'components', item: 'error-message', example: 'default'}) }}
+Use an error message when a there is a validation error. Explain what went wrong and how to fix it.
+
+{{ example({group: 'components', item: 'error-message', example: 'default', html: true, nunjucks: true, open: false}) }}
 
 ## When to use this component
 
-Show an error message next to the form input when there is a validation error with the information a user has entered.
+Show an error message next to the field and in the [Error summary](../error-summary/) when there is a validation error.
```

Comment:
Change:

[Error summary]

to:

[error summary]

Reply:
Updated to use lowercase everywhere we have a component name in this PR